.env.production

```diff
@@ -1 +1,2 @@
 REACT_APP_ARCHIVE_URL=https://archive.cnx.org/contents/
+SERVER_PORT=8000
```

Comment:
we probably don't even need to specify this here

Reply:
oops, yep. it's not needed.